docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,311 @@
+
+**The `databricks-iris` starter is already configured to save logs in this location.** The configuration used to achieve this can be found in the `conf/base/logging.yml` file.
+
+### Package your project
+
+To package your Kedro project for deployment on Databricks, you must create a Wheel (`.whl`) file, which is a binary distribution of your project. In the root directory of your Kedro project, run the following command:
+
+```bash
+kedro package
+```
+
+This command generates a `.whl` file in the `dist` directory within your project's root directory.
+
+### Upload project data and configuration to DBFS
+
+```{note}
+A Kedro project's configuration and data do not get included when it is packaged. They must be stored somewhere accessible to allow your packaged project to run.
+```
+
+Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to a location accessible to Databricks. In this guide, we will store the data on the Databricks File System (DBFS).
+
+The Databricks Iris starter contains a [catalog](../../data/data_catalog.md#the-data-catalog) that is set up to access data stored in DBFS (`<project_root>/conf/`). You will point your project to use configuration stored on DBFS using the `--conf-source` option when you create your job on Databricks.
+
+There are several ways to upload data to DBFS: you can use the [DBFS API](https://learn.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/dbfs), the [`dbutils` module](https://docs.databricks.com/dev-tools/databricks-utils.html) in a Databricks notebook or the [Databricks CLI](https://docs.databricks.com/dev-tools/cli/dbfs-cli.html). In this guide, it is recommended to use the Databricks CLI because of the convenience it offers.
+
+- **Upload your project's data and config**: at the command line in your local environment, use the following Databricks CLI commands to upload your project's locally stored data and configuration to DBFS:
+
+```bash
+databricks fs cp --recursive <project_root>/data/ dbfs:/FileStore/iris-databricks/data
+databricks fs cp --recursive <project_root>/conf/ dbfs:/FileStore/iris-databricks/conf
+```
+
+The `--recursive` flag ensures that the entire folder and its contents are uploaded. You can list the contents of the destination folder in DBFS using the following command:
+
+```bash
+databricks fs ls dbfs:/FileStore/iris-databricks/data
+```
+
+You should see the contents of the project's `data/` directory printed to your terminal:
+
+```bash
+01_raw
+02_intermediate
+03_primary
+04_feature
+05_model_input
+06_models
+07_model_output
+08_reporting
+```
+
+## Deploy and run your Kedro project using the workspace UI
+
+To run your packaged project on Databricks, login to your Databricks account and perform the following steps in the workspace:
+
+1. **Create a new job**: In the Databricks workspace, navigate to the `Worfklows` tab and click `Create Job`:
```

Comment:
```suggestion
1. **Create a new job**: In the Databricks workspace, click the `New` button, then `Job`:
```

this is more accurate?

Reply:
You can do either of these 🙂 I changed this to:

"navigate to the `Worfklows` tab and click `Create Job` or click the `New` button, then `Job`".